### ISSUE TITLE
failing with `cargo-release`'s `pre-release-replacements`

### ISSUE THREAD

**Issue by pinage404** [nix-community/naersk#222]
Hello,

TL;DR How to use Naersk with a crate that use [`cargo-release`'s `pre-release-replacements`](https://github.com/crate-ci/cargo-release/blob/master/docs/faq.md#how-do-i-update-my-readme-or-other-files) ?

---

to release a crate, i use [`cargo-release`](https://github.com/crate-ci/cargo-release)

to automatically update some files while releasing, i am using [`cargo-release`'s `pre-release-replacements`](https://github.com/crate-ci/cargo-release/blob/master/docs/faq.md#how-do-i-update-my-readme-or-other-files)

i am trying to Nixify this crate with Nix Flake and Naersk

but it fails to parse `Cargo.toml` with this error

```
cargo build $cargo_release -j "$NIX_BUILD_CORES" --message-format=$cargo_message_format
error: failed to parse manifest at `/build/dummy-src/Cargo.toml`

Caused by:
  could not parse input as TOML

Caused by:
  expected an equals, found a colon at line 7 column 64
[naersk] cargo returned with exit code 101, exiting
```

to try by yourself, i isolated the problem with [a test](https://github.com/pinage404/naersk/commit/f331531c1d2edf6387e61f96aa89208161ca00b0#diff-72b69942d0c2c4bb45957790c3afc7920d590bb8da60336764aec4209287fa7e)
